edgePy/DGEList.py

```diff
@@ -390,8 +409,7 @@ def create_DGEList_handle(
         # duplicate gene name, due to a putative bug in genfromtxt
         genes = genes[1:]
 
-        with smart_open(group_file, 'r') as gr:
-            group = json.load(gr)
+        group = json.load(group_handle)
```

Comment:
Would it not be best to keep to smart_open for if the groups file is on a server somewhere? Of course, this file is very small or will likely be made manually, so maybe not a priority?

Reply:
both options exist - no change needed.